packages/components/inputs/search-select-input/src/search-select-input-option.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
Are these just being used in the Examples for Storybook?
If so, I think we should put "example" on the filename & component name and/or leave a comment about it, just so it's clearer that this is not actually part of the component itself 🙂

Reply:
I want to showcase it to the designers first that the `Option` component is extendable and not sure if we need to provide the option of choosing between which `Option` component to use, since the use-cases differs a lot. I will move to either the story itself or rename the file after the demo.

Reply:
Sounds good 😄 

It could actually be nice to export them as pre-made styled options :)

Reply:
Right now they are not exported outside the component, but we could put it in the `select-utils` and export it there and then use it back here. 
p.s. follow this conversation as well https://github.com/commercetools/ui-kit/pull/1542/files#r484823827